### ISSUE TITLE
Unable to use bold+italic font in LuaLaTeX / LuaTeX

### ISSUE THREAD

**Issue by lunarythia** [psb1558/Elstob-font#35]
I have the variable font ttf version installed and LuaTeX should support those.

MWE:
```latex
\documentclass[14pt]{article}
\usepackage{fontspec}
\setmainfont[BoldFont=*,
BoldFeatures={RawFeature={axis={wght=700}}},
ItalicFont=Elstob-Italic,
BoldItalicFont=Elstob-Italic,
BoldItalicFeatures={RawFeature={axis={wght=700}}}]{Elstob}
\begin{document}
normal \textbf{bold} \textit{italic} \textbf{\textit{bolditalic}}
\end{document}
```

Error:
```
ERROR: Font \TU/Elstob(0)/b/it/10=Elstob-Italic:mode=node;script=latn;language=dflt;+tlig;axis={wght=700}; at 10pt not loadable: metric data not found or bad.

--- TeX said ---
<to be read again> 
relax 
l.10 ...\textit{italic} \textbf{\textit{bolditalic}}
                                                  
--- HELP ---
You requested a family/series/shape/size combination that is totally
unknown.  There are two cases in which this error can occur:
  1) You used the \size macro to select a size that is not available.
  2) If you did not do that, go to your local `wizard' and
     complain fiercely that the font selection tables are corrupted!
```

**Comment by lunarythia** [comment]
If I go without the wght=700 part, then the italic doesn’t show up.

**Comment by lunarythia** [comment]
![圖片](https://github.com/psb1558/Elstob-font/assets/63614345/64f8d489-3c7f-49b2-9667-d4df8ec6ecf4)
Normal, bold, and italic all render correctly. However, bold+italic, as you can see here, doesn’t show up at all.

**Comment by psb1558** [comment]
I'll fiddle with this a bit and try to find out what's going on.

**Comment by lunarythia** [comment]
Are you able to reproduce the error?

I only have the variable `Elstob.ttf` and `Elstob_Italic.ttf` installed.

**Comment by psb1558** [comment]
If I add `RawFeature={mode=harf}` to the `\setmainfont` command, then everything works properly. This is with `Elstob.ttf` and `Elstob-Italic.ttf` installed as local fonts (in `~/Library/Fonts`) on a Mac OS (Ventura) system.
```
\setmainfont[BoldFont=*,
BoldFeatures={RawFeature={axis={wght=700}}},
ItalicFont=Elstob-Italic,
BoldItalicFont=Elstob-Italic,
BoldItalicFeatures={RawFeature={axis={wght=700}}},
RawFeature={mode=harf}]{Elstob}
```
<img width="220" alt="image" src="https://github.com/psb1558/Elstob-font/assets/16215767/7a98e7e6-14d1-4417-922b-2a7f080420a3">

I don't know what version of LuaTeX you're using, but perhaps it's worth mentioning that every version of luaotfload up to the one packaged with TeXLive 2023 had significant, sometimes disabling bugs. (It may even be that I updated that program after I updated TeXLive this spring--I'm not sure.) The current version works flawlessly, but it is absolutely necessary to run in "harf" mode (using HarfBuzz for layout and much of the font management) rather than LuaTeX's native mode.

You might also want to look at [ElstobManual.sty](https://github.com/psb1558/Elstob-font/blob/master/docs/ElstobManual.sty) in the docs directory. It demonstrates a technique, based on a user's suggestions, for getting the Optical Size and other axes set properly for each point-size in your document.
